src/Web.cpp

```diff
@@ -350,11 +351,13 @@ void webserverStart(void) {
 			request->redirect("https://espuino.de/espuino/favicon.ico");
 		});
         // Init HallEffectSensor Value
-        #ifdef HALLEFFECT_SENSOR_ENABLE
+        #ifndef HALLEFFECT_SENSOR_ENABLE
```

Comment:
Why are you changing the preprocessor logic here?

Reply:
You are reverting this change in a later commit. Would be good if you could remove it from commit https://github.com/biologist79/ESPuino/pull/223/commits/475ba3776af09705898bcb25928c2f4a5e0a3a7d in the first place.

Reply:
yes, I had changed the ifdef to test the code and missed it in the first commit. It's now fixed.